lib/matplotlib/backend_tools.py

```diff
@@ -399,24 +399,74 @@ def trigger(self, sender, event, data=None):
                 a.set_navigate(i == n)
 
 
-class ToolGrid(ToolToggleBase):
-    """Tool to toggle the grid of the figure"""
+class _ToolGridBase(ToolBase):
+    """Common functionality between ToolGrid and ToolMinorGrid.
```

Comment:
I find this form of docstring formatting to be annoying. Have either both of the triple quotes on the same line, or each on their own line.